docs/contributing/core/index.md

```diff
@@ -0,0 +1,310 @@
+They will not work with buildout.
+To contribute to Volto, you will need to start over, and follow {doc}`../volto`.
+```
+
+To login, the default credentials are the following.
+
+-   username: `admin`
+-   password: `admin`
+
+
+## Work with git
+
+```{important}
+This section applies to members of the GitHub `plone/developers` team, who have write access to repositories under the Plone GitHub organization.
+
+Members of the `plone/contributors` team do not have write access, and instead must follow the process to set up their remote upstream and origin branches as described in {ref}`set-up-your-environment-label`.
+```
+
+Always begin by checking out the git branch on which you want to work.
+If you have not yet checked out a branch, then you need to track it with the `-t` option and specify the remote to track, usually called `origin`.
+The following command will switch and track changes from the remote `origin` and its branch `6.1`.
+
+```shell
+git checkout -t origin/6.1
+```
+
+Now going forward, you need to specify only the local branch when you want to switch back to it from your development branch.
+
+```shell
+git checkout 6.1
+```
+
+Next pull down and merge any recent changes from the remote tracked repository with a single command.
+
+```shell
+git pull
+```
+
+
+## Edit packages
+
+First identify the names of the Plone packages you want to work on.
+If you do not know, you can ask in the [Plone Community Forum](https://community.plone.org/).
+Only a few packages are in {file}`src/` by default.
+
+Next create a new file {file}`buildout.local.cfg`, and add the names of packages that you want to develop under the `auto-checkout` list.
+
+```cfg
+[buildout]
+extends =
+  buildout.cfg
+
+auto-checkout =
+    # Add packages that you want to develop
+    plone.app.event
+    icalendar
+    # others
+    ...
+```
+
+When you make changes in your package, then rerun buildout with the following command, specifying your new buildout configuration file with the `-c` option.
+You can add the `-N` flag to save time by not checking PyPI to see if there are updates to packages that were already installed.
+
+```shell
+./bin/buildout -c buildout.local.cfg -N
+```
+
+```{seealso}
+For tips on working with `mr.developer`, see {doc}`mrdeveloper`.
+```
+
+````{tip}
+To avoid conflicts with `buildout.coredev` files, you can configure git for your user.
+Either create or edit a file at {file}`~/.gitconfig`.
+Then add the following stanza to it.
+
+```cfg
+[core]
+    excludesfile = ~/.gitignore_global
+```
+
+Then add any standard `.gitignore` syntax to exclude files from getting committed and pushed to a remote repository.
+````
+
+Next create a new development branch on which you want to work from the current branch, tracking the upstream Plone repository, and check it out.
+It's a good idea to use a branch name that includes the issue number and is descriptive of what it resolves.
+
+```shell
+git switch -c 123-thing-i-fixed -t origin/6.1
```

Comment:
```suggestion
git switch -c 123-thing-i-fixed
```

`origin/6.1` only makes sense for buildout.coredev itself, which is not where we're creating this branch. And we don't want the new branch to track an existing remote branch -- the upstream branch it should track won't exist on the remote until it is pushed.

Reply:
Included with 26da8cc1d6fed9aa0adb12611384c29546c77e9e.